package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "./node_modules/.bin/eslint --format codeframe .",
-    "test": "./node_modules/.bin/mocha ./tests/tests.js"
+    "test": "./node_modules/.bin/nyc ./node_modules/.bin/mocha ./tests/tests.js"
```

Comment:
You made a `.nycrc` file but never added the config to that or to `package.json`.